core/src/test/java/org/hibernate/ogm/backendtck/associations/storageconfiguration/AssociationStorageConfiguredProgrammaticallyTest.java

```diff
@@ -31,7 +31,7 @@
  * @author Gunnar Morling
  */
 @SkipByGridDialect(
-		value = { GridDialectType.EHCACHE, GridDialectType.HASHMAP, GridDialectType.INFINISPAN, GridDialectType.NEO4J, GridDialectType.CASSANDRA },
+		value = { GridDialectType.EHCACHE, GridDialectType.HASHMAP, GridDialectType.INFINISPAN, GridDialectType.NEO4J, GridDialectType.CASSANDRA, GridDialectType.REDIS },
```

Comment:
2 reasons: I do now know, whether I have to implement sth to support programatically config and the test fails with a class cast exception: 

```
org.hibernate.ogm.datastore.redis.options.navigation.impl.RedisGlobalContextImpl_$$_jvst662_2 cannot be cast to org.hibernate.ogm.datastore.document.options.navigation.DocumentStoreGlobalContext
```

I can see I if can fix the `ClassCastException`by using a different GlobalContext type.